content/english/supply-chain-analytics/Challenge.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Extra Challenge"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 5
+---
+
+
+One of the fences broke loose in the farm and all the chickens and hens escaped! Because of this, we’re short on eggs and we now have to change the max capacity of eggs from 24 to 15. ☹ 
```

Comment:
```suggestion
One of the fences have broken loose in the farm and all the chickens and hens escaped! Because of this, we’re short on eggs and we now have to change the maximum capacity of eggs from 24 to 15. ☹ 
```